articles/index.md

```diff
@@ -68,37 +68,37 @@ Develop a [theory of change](/theory) for your activist organisation using our t
 
 <action-smart-large/>
 
-## We work with data & revolutionary tech
+## We are working with data & revolutionary technologies
 
 We don’t just write guides. Activist Handbook uses a **data-driven approach** and develops **digital tools** to empower change-makers.
 
 ### Large numbers, big impact
 
 - **🔍 1K+ search queries analysed:** We know what challenges activists face most, and what words they type into Google to find help. With this SEO data, we can write the guides that activists actually need and make sure our platform is found by the people looking for it.
 - **📚 4.7K resources:** We did not write our 450+ guides from scratch. Instead, we brought together material from experienced campaigned around the globe and organised all of it in into one single comprehensive handbook.
-- **🔥 4K+ activist trained every month:** Our number of volunteers can be counted on one hand and budget is €0. Yet, we manage to train thousands of activists every single month. And the more activists use and contribute to our wiki platform, the better it gets.
+- **🔥 4K+ activist trained every month:** Our number of volunteers can be counted on one hand and budget is €0. Yet, we manage to train thousands of activists every single month. Every person counts, and the more activists use and contribute to our wiki platform, the better it gets. Share your knowledge with others [right now](https://edit.activisthandbook.org/new)!
 
 ### A unique digital toolset
 
-- **🧑‍💻 We develop our own digital tools:** If we need it, we build it. We bring together our experience with web development and campaigning.
-- **💡 We empower activists with AI:** Our writers never have to start with an empty page. Using artifical intelligence, we generate a first draft, as well as illustrations to accompany the guide. Whatever tech the capitalist elite can get their hands on to protect their interests, we use it too and fight back.
-- **⚡️ We prioritise speed & accessibility:** Everyone should be able to use Activist Handbook, no matter if you are on a slow or expensive data plan. So we put in the work to make that possible. And it shows. For example, [our platform](https://gtmetrix.com/reports/new.activisthandbook.org/14N3WvPa/) loads 8x faster and uses 87% less data than the [Beautiful Trouble](https://gtmetrix.com/reports/beautifultrouble.org/AR67fKWP/) website (tested 25 dec 2022).
+- **🧑‍💻 We develop our own digital tools:** If we need it, we build it. We bring together our experience with campaigning and web development.
+- **💡 We empower activists with AI:** Writing whole articles seems daunting for anyone, but you will never have to worry about where to start, as we will help you with AI text generation. You can also easily generate pictures to accompany your guide. Inspiring others is an art, and you are the artist!
```

Comment:
I am not sure about removing the capitalist part, but it didn't make sense with the new text, and I thought that focusing on inspiring others was better in this case.

Reply:
This is not on our home page anymore. But you might find this article interesting, which elaborates on how exactly we use AI:
https://activisthandbook.org/contribute/write/ai